template/App.tsx

```diff
@@ -35,7 +36,7 @@ const App = () => {
           contentInsetAdjustmentBehavior="automatic"
           style={styles.scrollView}>
           <Header />
-          {global.HermesInternal == null ? null : (
+          {!usingHermes ? null : (
```

Comment:
Isn't it enough to just remove the global prefix? I would prefer to not change too much about the logic here.

Reply:
No, because Jest fails without it. It was actually there but it was removed https://github.com/react-native-community/react-native-template-typescript/pull/55

Reply:
Just tested here on a new project on Android/IOS and it works fine, without throwing any errors at ```global```